homeassistant/components/sensor/sytadin.py

```diff
@@ -0,0 +1,83 @@
+"""
+Support for Sytadin Traffic, French Traffic Supervision.
+
+Systadin website : http://www.sytadin.fr
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.sytadin/
+"""
+import logging
+
+from homeassistant.const import LENGTH_KILOMETERS
+from homeassistant.helpers.entity import Entity
+
+_LOGGER = logging.getLogger(__name__)
+
+REQUIREMENTS = ['lxml', 'requests']
+
+SYSTADIN = 'http://www.sytadin.fr/sys/barometres_de_la_circulation.jsp.html'
+
+TRAFFIC_JAM_XPATH = '//*[@id="main_content"]/div[1]/div/span[3]/text()'
+MEAN_VELOCITY_XPATH = '//*[@id="main_content"]/div[2]/div/span[3]/text()'
+CONGESTION_XPATH = '//*[@id="main_content"]/div[3]/div/span[3]/text()'
+
+TRAFFIC_JAM_REGEX = '([0-9]+)'
+MEAN_VELOCITY_REGEX = '([0-9]+)'
+CONGESTION_REGEX = '([0-9]+.[0-9]+)'
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Setup the sensor platform."""
+    add_devices([
```

Comment:
Please allow the users to choose which sensors they would like to show like `monitored_conditions` in other platforms.